job_extra_field.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I might miss some context, where is this file used? Thanks!

Reply:
Oh, this is just a test input file. See how I used it:

https://paste.googleplex.com/5997243083784192